pycheribuild/projects/project.py

```diff
@@ -2573,7 +2573,7 @@ def configure(self, **kwargs):
             # CMAKE_CROSSCOMPILING will be set when we change CMAKE_SYSTEM_NAME:
             # This means we may not need the toolchain file at all
             # https://cmake.org/cmake/help/latest/variable/CMAKE_CROSSCOMPILING.html
-            system_name = "Generic" if self.target_info.is_baremetal else "FreeBSD"
+            system_name = "Generic" if self.target_info.is_baremetal or self.target_info.is_rtems else "FreeBSD"
```

Comment:
This is fine for now, but longer-term we should add `self.target_info.cmake_system`